src/nvim/edit.c

```diff
@@ -3815,8 +3817,12 @@ static void ins_compl_delete(void)
   // TODO: is this sufficient for redrawing?  Redrawing everything causes
   // flicker, thus we can't do that.
   changed_cline_bef_curs();
+  // clear v:completed_item
+  set_vim_var_dict(VV_COMPLETED_ITEM, dict_alloc());
 }
 
+#define SHOULD_BE_STR(x) ((x) ? (x) : (char_u *)"")
```

Comment:
Is there not already a macro in `macros.h` for this? If not, perhaps it should be added there. Perhaps a name like `EMPTY_IF_NULL` would be better.